### DISCUSSION TITLE
Public Dashboard Variable Issue

### DISCUSSION THREAD

**Discussion by mrdeadlocked** [grafana/grafana-infinity-datasource#724] [Q&A]
Good Morning,

I am working on a dashboard pulling Snipe-IT Data (REST API).

Company Variable Set
![image](https://github.com/grafana/grafana-infinity-datasource/assets/461352/e62d644e-ee32-4283-94ff-1d9351acb4d4)
![image](https://github.com/grafana/grafana-infinity-datasource/assets/461352/643ca2bf-10bf-48ca-b58c-0da336c0c687)

The variable works great on a normal dashboard.
![image](https://github.com/grafana/grafana-infinity-datasource/assets/461352/de32f97d-653e-4f00-8dc9-ab078ee0ec04)

My problem is that when I try to use it on a public dashboard I get this.
![image](https://github.com/grafana/grafana-infinity-datasource/assets/461352/a1e763d0-8ec4-491a-a766-97cc3020e9e5)

Here is what the console shows
![image](https://github.com/grafana/grafana-infinity-datasource/assets/461352/0c6130c1-46ac-49f7-8c8b-805558314661)

Is this something with the data source or something else that I am missing?

**Comment by yesoreyeram** [comment]
As per grafana documentation, variables are not supported in public dashabord. Look out the Limitations section in https://grafana.com/docs/grafana/latest/dashboards/dashboard-public/.

**Comment by mrdeadlocked** [comment]
Alright. Sorry about that. Thank you!
